Naming Cleanup

deccer opened the pull request:
Match naming class/file
Fix typos
Pull types out into folders f.i. Renderer/Batching contains all commands

deccer committed:
Add UO to shared dictionary

deccer committed:
Naming cleanup

Match naming class/file
Fix typos
Pull types out into folders f.i. Renderer/Batching contains all commands

andreakarasho commented on src/Network/Plugin.cs:
```diff
@@ -681,8 +682,11 @@ private static bool UnblockFile(string fileName)
+            GraphicsDevice device,
+            IntPtr ptr,
+            int length,
+            IDictionary<IntPtr, GraphicsResource> resources)
```
Why IDictionary and not a Dictionary class directly?

deccer replied:
that way other `IDictionary<TKey, TValue>` impls can be passed in. SortedDict/ConcurrentDict/MyFancyDict....

andreakarasho commented on src/Configuration/Profile.cs:
```diff
@@ -527,31 +527,31 @@ public List<Gump> ReadGumps()
-                            int x = int.Parse(xml.GetAttribute("x"));
-                            int y = int.Parse(xml.GetAttribute("y"));
-                            uint serial = uint.Parse(xml.GetAttribute("serial"));
+                            GumpType type = (GumpType) int.Parse(xml.GetAttribute(nameof(type)));
```
Why did you prefer the nameof approach? Not because it’s wrong, i’m not a fan of floating strings. Im just curious

deccer replied:
Hmm looks like i forgot the other 3 hehe.

nameof() is just compiletime and the name is really the same, renaming stuff is easier for most IDEs - especially a VS without Resharper

deccer committed:
Fix protected field naming style

deccer committed:
Further renaming refactorings

totalMS -> totalTime
frameMS -> frameTime
ProfileManager.Current -> ProfileManager.CurrentProfile
Texture2DCache -> SolidColorTextureCache
protected fields, corrected naming style
pulled classes out into single files

deccer committed:
Re-add FNA.Core
